partcad/src/partcad/actions/part_actions.py

```
@@ -0,0 +1,142 @@
+import shutil
+import copy
+from pathlib import Path
+import partcad.logging as pc_logging
+from partcad.project import Project
+from partcad.utils import resolve_resource_path
+
+def add_part_action(project, kind, path, config=None):
```

Ruff (0.8.2)

Missing return type annotation for public function `add_part_action`

Add return type annotation: `None`

(ANN201)